pkg/resource/model_package_group/testdata/v1alpha1/create/desired/success_before_create.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: ModelPackageGroup
+metadata:
+  name: xgboost-model-package-group
+spec:
+  modelPackageGroupName: xgboost-model-package-group
+  modelPackageGroupDescription: "Description for model package group"
+  tags:
+    - key: algorithm
+      value: xgboost
+    - key: environment
+      value: testing
+    - key: customer
+      value: test-user
```

Comment:
nit : do we need all these tags in unit test 
in every unit test ?

Reply:
Helps with coverage in sdk.go create but I guess not necessary for all tests